.github/workflows/tests.yml

```diff
@@ -0,0 +1,18 @@
+name: Tests
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: ["2.7", "3.6", "3.7", "3.8", "3.9", "3.10"]
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+      - run: pip install .[test]
+      - run: pytest
```

Comment:
Built green in my fork
https://github.com/AndreMiras/lru-dict/actions/runs/2414070733